stylesheets/commons/Fountain.less

```diff
@@ -0,0 +1,1756 @@
+	image-rendering: -moz-crisp-edges;
+	image-rendering: -o-crisp-edges;
+	image-rendering: -webkit-optimize-contrast;
+	image-rendering: crisp-edges;
+	image-rendering: pixelated;
+	-ms-interpolation-mode: nearest-neighbor;
+}
+
+/* Image links ([[Template:Ability ID]], [[Template:Hero ID]], ...) */
+.image-link {
+	white-space: nowrap;
+}
+
+/*
+* Animatrion styles ([[Template:Animate]])
+*/
+
+.animated > *:not( .animated-active ),
+.animated-paused > *:not( .animated-active ),
+.animated > .animated-subframe > *:not( .animated-active ),
+.animated-paused > .animated-subframe > *:not( .animated-active ) {
+	display: none;
+}
+
+/* Hero infobox styles ([[Template:Hero infobox]]) */
+
+/* Highlight the primary attribute icon */
+#primaryAttribute img,
+.primaryAttribute img {
+	border: 2px solid #ffffff;
+	border-radius: 50%;
+}
+
+/*
+* Cosmetic Item infobox styles ([[Template:Cosmetic item infobox]])
+*/
+
+.buynow_wrapper {
+	height: 24px;
+}
+
+.buynow_wrapper > a {
+	width: 100%;
+	color: #ffffff;
+	text-decoration: none;
+}
+
+.buynow_button {
+	display: block;
+	padding-left: 30px;
+	height: 24px;
+	background: linear-gradient( 0deg, #000000 0%, #666666 100% );
+	color: #ffffff;
+	text-align: left;
+	text-decoration: none;
+	font-size: 90%;
+	line-height: 24px;
+}
+
+.buynow_button:hover {
+	background: linear-gradient( 0deg, #000000 0%, #999999 100% );
+}
+
+.buynow_store:before {
+	position: absolute;
+	margin-left: -31px;
+	width: 24px;
+	height: 24px;
+	// TODO: Steam icon
+	content: "";
+	pointer-events: none;
+}
+
+.buynow_market:before {
+	position: absolute;
+	margin-left: -31px;
+	width: 24px;
+	height: 24px;
+	// TODO: Steam icon
+	content: "";
+	pointer-events: none;
+}
+
+/*
+* Patch styles ([[Template:Patch layout]])
+*/
+
+html.theme--dark .patch-layout {
+	border: 1px solid #555555;
+}
+
+.patch-layout {
+	width: 100%;
+	border: 1px solid #aaaaaa;
+	border-collapse: unset;
+	text-align: center;
+}
+
+html.theme--dark .patch-layout-last {
+	background: #202d78;
+}
+
+.patch-layout-last {
+	background: #b2d7e9;
+}
+
+html.theme--dark .patch-layout-all {
+	background: #313c7e;
+}
+
+.patch-layout-all {
+	width: 27%;
+	background: #dee8ed;
+}
+
+html.theme--dark .patch-layout-background {
+	background-color: rgba( 41, 41, 41, 0.7 );
+	border: 1px solid #555555;
+}
+
+.patch-layout-background {
+	background-color: #fdfaf8;
+	border: 1px solid #aaaaaa;
+	margin: 0.75em 0;
+	padding: 0.75em 1em;
+	padding-bottom: 1em;
+}
+
+/*
+* Event Currency Colors
+*/
+
+.bp2020 {
+	color: #f0b000;
+}
+
+.bp2019 {
+	color: #8c1bcd;
+}
+
+.bp2018 {
+	color: #3ec415;
+}
+
+.bp2017 {
+	color: #08d8d8;
+}
+
+.bp2016 {
+	color: #e60000;
+}
+
+.dotaplus {
+	color: #d37708;
+}
+
+/* AudioButton extension styles */
+
+a.ext-audiobutton:before {
+	display: inline-block;
+	width: 1em;
+	vertical-align: middle;
+}
+
+a.ext-audiobutton {
+	/* stylelint-disable-next-line font-family-no-missing-generic-family-keyword */
+	font-family: "Font Awesome 5 Pro";
```

Comment:
unsure if we should just ignore it or if we want to add the font as supported or something else^^

Reply:
@FO-nTTaX @liquidely 

- [ ] Ignore line
- [ ] [Rule exception](https://stylelint.io/user-guide/rules/font-family-no-missing-generic-family-keyword/#ignorefontfamilies-regex-regex-string)
- [ ] [Fallback font](https://stackoverflow.com/a/67738590)

Which option would you prefer?

Reply:
I'd say comment it out/remove it. We can always add it back when needed. We have font awesome support on the website so it feels unnecessary?

Reply:
If I'm not mistaken, it's needed for the audio buttons from their extension to have a FA icon

Reply:
Alright then I would go for 'disable-next-line'